template/build/utils.js

```diff
@@ -25,6 +25,8 @@ exports.cssLoaders = function (options) {
       return loader + (options.sourceMap ? extraParamChar + 'sourceMap' : '')
     }).join('!')
 
+    // Extract CSS when that option is specified
+    // (which is the case during production build)
```

Comment:
Note: this is not *really* related to this PR, just some added commenting, might help someone reading it.